--- origsrc/libpgf/doc/Doxyfile.in	2009-10-01 12:07:02.000000000 -0500
+++ src/libpgf/doc/Doxyfile.in	2010-09-08 17:23:53.582176100 -0500
@@ -193,7 +193,7 @@ WARN_LOGFILE           =
 #---------------------------------------------------------------------------
 # configuration options related to the input files
 #---------------------------------------------------------------------------
-INPUT                  = ../include/
+INPUT                  = @top_srcdir@/include/
 INPUT_ENCODING         = UTF-8
 FILE_PATTERNS          = *.c \
                          *.cc \
